printer_mods/README.md

```diff
@@ -125,7 +125,7 @@ like so:
 | | [V0 beltchains](./Iakabos/V0_beltchain)|Modified beltchain parts for V0 Z cables|:heavy_check_mark: |:x: |:x: |:x:|
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
 |jeoje |[V2.4 Z Chain Guide with Integrated Chamber Thermistor Mount](./jeoje/Z_Chain_Guide_Thermistor_Mount) | Modifies the Z chain guide to hold a chamber thermistor  |:x: |:x: |:heavy_check_mark: |:x:|
-| | [V2.4 4.3" Touchscreen Mount](./jeoje/4.3_Inch_Touchscreen_Mount) | Mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) to v2.4 Vorons |:x:|:x:|:heavy_check_mark:|:x:|
+| | [V2.4 4.3" Touchscreen Mount](./jeoje/4.3_Inch_Touchscreen_Mount) | Mounts a [Waveshare 4.3inch Capacitive Touch Display](https://www.waveshare.com/4.3inch-dsi-lcd.htm) to v2.4 Vorons |:x:|:x:|:heavy_check_mark:|:heavy_check_mark:|
```

Comment:
If this is now compatible with a Switchwire, you should modify the title and description here to remove V2.4.

Reply:
I don't have a problem with this. Thanks for the ping. Also, that part of the logo prints 1 layer off the bed intentionally.

Reply:
i've updated the readme
sorry for not pinging you @jeoje and thanks for the allowance